Octokit/Helpers/HttpClientExtensions.cs

```diff
@@ -7,16 +7,12 @@ namespace Octokit
 {
     public static class HttpClientExtensions
     {
-        /// <summary>
-        /// OBSOLETE
-        /// </summary>
-        [Obsolete("This will be removed in a future release")]
-        public static Task<IResponse<T>> Send<T>(this IHttpClient httpClient, IRequest request)
+        public static Task<IResponse> Send(this IHttpClient httpClient, IRequest request)
```

Comment:
Yeah, this bit me upgrading to vLatest this week. 

I didn't mind calling out `CancellationToken.None` for these few usages, so I guess I'm debating cosmetics and shortcuts here. Happy to bring it back :tada: